resources/todomvc/big-dom-generator/src/app.css

```diff
@@ -86,6 +86,22 @@ Importing the CSS from the Spectrum CSS library.
     min-width: 30px;
 }
 
+.display-none {
+    display: none;
+}
+
+/*
+The spectrum rule making the open tree view items visible is
+.spectrum-TreeView-item.is-open > .spectrum-TreeView {
+    visibility: visible;
+}
```

Comment:
Can you add a few additional lines explaining that the normal TreeView has `visibility: hidden`?

Then explain that `unset` has the same effect as `visible` for the visible nodes, but doesn't have the side effect of showing descendants?

Reply:
Done.